### DISCUSSION TITLE
Ability to use private STUN/TURN server

### DISCUSSION THREAD

**Discussion by GeghamSimonyan** [OvenMediaLabs/OvenMediaEngine#1716] [Ideas] [answered]
Good day
would be nice to add auth section for TURN/STUN servers to allow use private servers like **twilio**

**Comment by getroot** [comment]
OvenMediaEngine does not use stun/turn to establish ICE connection with peer. Stun setting in OME's Server.xml is not related to Webrtc and is only used to find out its own IP.

Players can set up a private stun/turn server, but they won't need to do that when receiving streams from OME. OME allows players to receive streams without using a Stun/Turn server.

**Comment by nums** [comment, marked as answer]
You can setup OME with 443 port and force TCP for webRTC
But if it is an absolutely need, you can set up this on player side : https://airensoft.gitbook.io/ovenplayer/initialization#webrtcconfig.iceservers
